src/main/osd/osd_warnings.c

```diff
@@ -248,6 +249,15 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
 
 #endif // USE_GPS_RESCUE
 
+#ifdef USE_POS_HOLD_MODE
+    if (osdWarnGetState(OSD_WARNING_POSHOLD_FAILED) && posHoldFailure()) {
+        tfp_sprintf(warningText, "POS HOLD FAIL");
```

Comment:
OK then, best to wait until 14057 is merged.  I haven't had any issues despite using this warning and seeing it in the OSD when required.

Reply:
#140457 affects only MSP osd

This is 1 byte overflow, it may overwrite something that does not have immediate consequences. In this particular case, it may overlap with `displayAttr`, so string terminator may be removed (displayAttr is written after string) and random data may be displayed (next byte is nonzero `blinking`,  saved registers may follow).

Larger overflow will be much more dangerous - working register overwrite - any behavior possible, then return address (saved LR) -  probable hardfault.